src/ca/team2706/vision/trackerboxreloaded/Main.java

```diff
@@ -348,29 +468,36 @@ public static void main(String[] args) {
                     continue;
                 }
             }
-
-
             // log images to file once every seconds_between_img_dumps
             long elapsedTime = (System.currentTimeMillis() / 1000) - current_time_seconds;
+            //If the elapsed time is more that the seconds between image dumps then dump images asynchronously
             if (elapsedTime >= seconds_between_img_dumps) {
+            	//Sets the current number of seconds
                 current_time_seconds = (System.currentTimeMillis() / 1000);
-
-                Mat finalFrame = frame.clone();
+				//Clones the frame
+				Mat finalFrame = frame.clone();
+				//Starts a new thread to dump images
                 new Thread(new Runnable() {
                     public void run() {
                         try {
+                        	//Dumps the raw image
                             imgDump(matToBufferedImage(finalFrame), "raw");
+                            //Dumps the binMask image
                             imgDump(matToBufferedImage(visionData.binMask), "binMask");
-                            imgDump(matToBufferedImage(outputTargetImg), "output");
+                            //Draw the target to the output image
+                            Pipeline.drawPreferredTarget(finalFrame, visionData);
+                            //Dumps the output image
+                            imgDump(matToBufferedImage(finalFrame), "output");
                         } catch (IOException e) {
```

Comment:
Another code change. Didn't `outputTargetImg` already had the targets drawn on it from [here](https://github.com/FRC2706/TrackerboxReloaded/pull/57/files#diff-051c1874533492a85443dcaa4edd1192L326)?